modules/@apostrophecms/schema/ui/apos/components/AposInputAttachment.stories.js

```diff
@@ -0,0 +1,55 @@
+import {
+  withKnobs,
+  select,
+  boolean
+} from '@storybook/addon-knobs';
+
+export default {
+  title: 'Input (Attachment)',
+  decorators: [ withKnobs ]
+};
+
+export const attachmentInput = () => {
+  const isDisabled = boolean('Disabled', false);
+  const limit = select(
```

Comment:
Should all this limit stuff still be here?

Reply:
I don't know. If everyone agrees, I will remove it.

Reply:
The story should reflect the current capability. the multiple attachment knobs can go for now